docs/mobility/_index.md

```diff
@@ -2,11 +2,59 @@
 title: "Mobility"
 linkTitle: "Mobility"
 weight: 460
-simple_list: true
 type: "docs"
 description: "Plan and control your robot's motion with Viam's integrated tools."
 image: "/platform/mobility.svg"
 imageAlt: "A path around obstacles."
-empty_node: true
 menuindent: true
+no_list: true
 ---
+
+The {{< glossary_tooltip term_id="rdk" text="RDK" >}} includes several built-in services that make it easy to move your robot in intelligent ways, whether moving a single arm, a connected system of gantries and arms, or an entire robot on wheels.
+
+The following services work together to coordinate your robot's movement:
+
+## Frame system
+
+{{<imgproc src="/services/icons/frame-system.svg" class="fill alignleft" style="max-width: 190px" declaredimensions=true alt="Frame system icon">}}
+
+The [frame system service](/mobility/frame-system/):
+
+- Allows you to describe the physical relationship between your machine's parts by organizing the machine's {{< glossary_tooltip term_id="component" text="components" >}} into a tree of reference frames.
+- Displays a visualization of that tree of reference frames in <a href="https://app.viam.com">the Viam app</a>.
+- Has an API method that allows you to translate a point in the coordinate system of one component (such as a camera) to a point in the coordinate system of another component (such as an arm)
+- The motion service uses the tree of reference frames internally to make and execute motion plans while avoiding self-collisions as well as collisions with external objects.
+
+## Motion and navigation
+
+{{<imgproc src="/services/icons/motion.svg" class="fill alignleft" style="max-width: 170px" declaredimensions=true alt="Motion path icon">}}
```

Comment:
I think this is a nice touch but screenshots or project gifs would be more effective in conveying the information I think? Why not use the moving arm one with the cup? and one of a robot navigating on a gps map?

Reply:
The cup one is present at the bottom of the page in the tutorials--do you not think it's bad to have it twice? I suppose I could make a new one actually (and make it go faster).

I can add both--for me I don't default to doing so because adding multiple gifs makes the page distractingly busy and is thus harder to read (kind of like ad blocker vs not).

Reply:
I think fine to reuse, alternatively use the claw game one ;)